scripts/pipeline-launch/launch.sh

```diff
@@ -221,6 +221,19 @@ function cp_cap_publish {
             
             sed -i "/$_SGE_WORKER_INIT/d" $_WORKER_CAP_INIT_PATH
             echo "$_SGE_WORKER_INIT" >> $_WORKER_CAP_INIT_PATH
+
+            # Due to strange behavior of sge_execd with updating global config values,
+            # we need to change default mail client if favor to our custom
+            if check_cp_cap "CP_CAP_GRID_ENGINE_NOTIFICATIONS"
+            then
+                _PIPE_MAIL_ENABLER_SCRIPT="pipe_mail_enabler"
+
+                sed -i "/$_PIPE_MAIL_ENABLER_SCRIPT/d" $_MASTER_CAP_INIT_PATH
+                echo "$_PIPE_MAIL_ENABLER_SCRIPT" >> $_MASTER_CAP_INIT_PATH
+
+                sed -i "/$_PIPE_MAIL_ENABLER_SCRIPT/d" $_WORKER_CAP_INIT_PATH
+                echo "$_PIPE_MAIL_ENABLER_SCRIPT" >> $_WORKER_CAP_INIT_PATH
+            fi
```

Comment:
What do you think if we simplify it by moving this to the beginning of launch.sh?

```bash
if check_cp_cap CP_CAP_GRID_ENGINE_NOTIFICATIONS && check_cp_cap CP_CAP_SGE; then
    export CP_CAP_MAIL="true"
fi
```

Reply:
that was my idea too, but the thing is that in this case we won't be able to propagate it to workers, since CP_CAP_SGE will not be set to `true` for them